### ISSUE TITLE
The fundamental NGX component does not provide any inputs on tooltips

### ISSUE THREAD

**Issue by AnushkaAG29** [SAP/fundamental-ngx#12719]
#### Is this a bug, enhancement, or feature request?
Feature request

#### Describe your proposal.
The table components used from the NGX platform do not include tooltips for usage.

#### Can you handle that on the application side
--

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
0.53.0

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
In the table, there are drop-downs inside rows. A tooltip is required for those drop-downs. But there is no code which includes tooltips here.

#### Please provide relevant source code (if applicable).
HTML code:

 <fdp-table
      #table
      [dataSource]="source"
      [childDataSource]="childSource"
      [isTreeTable]="true"
      selectionMode="multiple"
      relationKey="children"
      emptyTableMessage="No data found"
      bodyHeight="300px"
      [virtualScroll]="true"
      [pageScrolling]="true"
      [pageSize]="50"
      [renderAhead]="50"
      [pageScrollingThreshold]="80"
      hasChildrenKey="hasChildren"
      [expandOnInit]="true"
 >
    <fdp-table-toolbar title="Order Line Items" [hideItemCount]="false" [hideSearchInput]="true"> </fdp-table-toolbar>

    <fdp-column
        name="name"
        key="name"
        label="Name"
        align="start"
        [sortable]="true"
        [filterable]="true"
        [dataType]="dataTypeEnum.STRING"
    >
    </fdp-column>

    <fdp-column name="description" key="description" label="Description"> </fdp-column>

    <fdp-column
        name="price"
        key="price.value"
        label="Price"
        align="end"
        [sortable]="true"
        [filterable]="true"
        [dataType]="dataTypeEnum.NUMBER"
    >
    </fdp-column>

    <fdp-column
        name="status"
        key="status"
        label="Status"
        align="center"
        [sortable]="true"
        [filterable]="true"
        [dataType]="dataTypeEnum.STRING"
    >
    </fdp-column>
</fdp-table>

<fdp-table-p13-dialog [table]="table"></fdp-table-p13-dialog>

#### Please provide stackblitz example(s).
--

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Since the NGX component doesn't include tooltips, it violates the ACC standard and it needs to be fixed. (As said by the accessibility expert).

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?
https://sap.github.io/fundamental-ngx/#/platform/table/advanced

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by mikerodonnell89** [comment]
What "drop downs"? Can you give more specific details about what is missing from where, and what you'd like to see?

**Comment by AnastasiyaVavilava** [comment]
I have the same issue with table action buttons that don’t have tooltips (ex. arrows in clickable rows)
<img width="290" alt="Screenshot 2024-11-26 at 12 53 18" src="https://github.com/user-attachments/assets/e7518d98-8d79-4e6d-9b95-d314bdacd0cf">
<img width="290" alt="Screenshot 2024-11-26 at 12 53 07" src="https://github.com/user-attachments/assets/31f034d1-b3b5-4c1e-9036-0c6f511d55ef">

**Comment by AnushkaAG29** [comment]
<img width="449" alt="Screenshot 2024-11-28 at 9 14 03 AM" src="https://github.com/user-attachments/assets/43b8458b-8d72-441c-870f-6c566331f904">
Tooltips are needed here for these drop downs under the name column.
